releases/2.13.11.md

```diff
@@ -0,0 +1,82 @@
+# Scala 2.13.11
+
+The Scala team at Lightbend is pleased to announce the availability of Scala 2.13.11.
+
+The following changes are highlights of this release:
+
+### Collections
+
+* Faster `Vector` concatenation ([#10159](https://github.com/scala/scala/pull/10159) by [@ansvonwa](https://github.com/ansvonwa))
+* New `LinkedHashMap`/`LinkedHashSet` implementation ([#10221](https://github.com/scala/scala/pull/10221) by [@liang3zy22](https://github.com/liang3zy22))
+
+### Compatibility
+
+* Support JDK 20 and 21 by upgrading to ASM 9.5 ([#10363](https://github.com/scala/scala/pull/10363), [#10184](https://github.com/scala/scala/pull/10184) by [@Philippus](https://github.com/Philippus))
+* Fix crash reading JDK 21 classfiles ([#10397](https://github.com/scala/scala/pull/10397))
+* Support Java 17 `sealed` in Java sources and classfiles ([#10348](https://github.com/scala/scala/pull/10348), [#10105](https://github.com/scala/scala/pull/10105) by [@som-snytt](https://github.com/som-snytt))
+
+### Align with Scala 3
+
+* Accept `implicit _`, `implicit (x: Int)` in lambdas, like Scala 3 does ([#10327](https://github.com/scala/scala/pull/10327) by [@som-snytt](https://github.com/som-snytt))
+* Inherited members no longer take precedence over outer definitions ([#10220](https://github.com/scala/scala/pull/10220), [#10339](https://github.com/scala/scala/pull/10339))
+* Deprecate infix type args, as they are dropped in Scala 3 ([#10255](https://github.com/scala/scala/pull/10255) by [@som-snytt](https://github.com/som-snytt))
+* Allow eta-expansion of methods with dependent types ([#10166](https://github.com/scala/scala/pull/10166))
+* Abstract parent `copy` does not suspend case `copy` ([#10085](https://github.com/scala/scala/pull/10085) by [@som-snytt](https://github.com/som-snytt))
+* Don't GLB binders of type patterns, use the type directly ([#10247](https://github.com/scala/scala/pull/10247))
+* Under `-Xsource:3`, adjust 2.13.9 change to ignore override type for whitebox macro expansion ([#10188](https://github.com/scala/scala/pull/10188) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, use narrowest type for pt of override  ([#10198](https://github.com/scala/scala/pull/10198) by [@som-snytt](https://github.com/som-snytt))
+* Under `-Xsource:3`, warn about change to parenless lambda parens ([#10320](https://github.com/scala/scala/pull/10320) by [@som-snytt](https://github.com/som-snytt))
```

Comment:
Could add a section just for `-Xsource:3`

### Migration to Scala 3 under `-Xsource:3`

or

### `-Xsource:3`: Mutant Scala

Reply:
meh... no strong opinion either way, but I think I like it a bit better as-is